src/components/TrophyHeader/TrophyHeader.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import UserIcon from "../UserIcon/UserIcon";
+import TrophyStats from "../TrophyStats/TrophyStats";
+import "./TrophyHeader.scss";
+
+const TrophyHeader = ({ userProfile }) => {
```

Comment:
I thought this was just a component tree fix? Catch up with you after lunch to discuss this?